libcxx/include/__config

```diff
@@ -174,11 +174,6 @@
 // The implementation moved to the header, but we still export the symbols from
 // the dylib for backwards compatibility.
 #    define _LIBCPP_ABI_DO_NOT_EXPORT_TO_CHARS_BASE_10
-// Save memory by providing the allocator more freedom to allocate the most
```

Comment:
We need to adjust the release note in LLVM 18 to say that this isn't only done in the unstable ABI anymore.

Reply:
Do you want this in the LLVM 18 release?

Reply:
Yeah, I think I would put it in LLVM 18. That way we'd never ship the ABI macro and we'd avoid potential user churn.

Reply:
Done.